src/schema/v2/home/__tests__/home_page_fairs_module.test.js

```diff
@@ -1,40 +1,79 @@
-/* eslint-disable promise/always-return */
 import { runQuery } from "schema/v2/test/utils"
+import moment from "moment"
+import { range } from "lodash"
+
+// adding this dep explicitly so the tests run when we change the main file
+;() => {
+  require("../home_page_fairs_module")
+}
+
+let _counter = 0
+const uniqid = () => _counter++
+
+const mockRunningFair = () => {
+  const id = uniqid()
+  return {
+    id: `running-fair-${id}`,
+    default_profile_id: `running-fair-${id}`,
+    start_at: moment().subtract(1, "day"),
+    end_at: moment().add(1, "day"),
+    name: `A running fair ${id}`,
+    published: true,
+    subtype: null,
+    summary: "",
+    layout: null,
+    display_vip: false,
+    has_full_feature: true,
+  }
+}
+
+const mockFutureFair = () => {
+  const id = uniqid()
+  return {
+    id: `future-fair-${id}`,
+    default_profile_id: `future-fair-${id}`,
+    start_at: moment().add(1, "day"),
+    end_at: moment().add(10, "day"),
+    name: `A future fair ${id}`,
+    published: true,
+    subtype: null,
+    summary: "",
+    layout: null,
+    display_vip: false,
+    has_full_feature: true,
+  }
+}
+
+const mockPastFair = () => {
+  const id = uniqid()
+  return {
+    id: `past-fair-${id}`,
+    default_profile_id: `past-fair-${id}`,
+    start_at: moment().subtract(10, "day"),
+    end_at: moment().subtract(1, "day"),
+    name: `A past fair ${id}`,
+    published: true,
+    subtype: null,
+    summary: "",
+    layout: null,
+    display_vip: false,
+    has_full_feature: true,
+  }
+}
+
+const isRunningFair = (fair) => fair.name.startsWith("A running fair")
+const isNotFutureFair = (fair) => !fair.name.startsWith("A future fair")
+
+const runFairsQuery = async (query, config) =>
+  runQuery(query, {
+    fairsLoader: (options) => Promise.resolve(config(options)),
+  })
 
-// FIXME: These tests seem to be failing in CI. Revisit and investigate why.
 describe("HomePageFairsModule", () => {
-  it.skip("works", () => {
-    const runningFairs = [
-      {
-        id: "artissima-2017",
-        default_profile_id: "artissima-2017",
-        start_at: "2017-11-03T10:00:00+00:00",
-        end_at: "2017-11-05T18:00:00+00:00",
-        name: "Artissima 2017",
-        published: true,
-        subtype: null,
-        summary: "",
-        layout: null,
-        display_vip: false,
-        has_full_feature: true,
-      },
-    ]
-
-    const pastFairs = [
-      {
-        id: "zonamaco-foto-and-sal-n-del-anticuario-2017",
-        default_profile_id: "zsonamaco-foto-2017",
-        start_at: "2017-09-20T13:45:00+00:00",
-        end_at: "2017-09-24T13:45:00+00:00",
-        name: "Zâ“ˆONAMACO FOTO & SALÃ“N DEL ANTICUARIO 2017",
-        published: true,
-        subtype: null,
-        summary: "",
-        layout: null,
-        display_vip: false,
-        has_full_feature: true,
-      },
-    ]
+  it("works", async () => {
```

Comment:
could we update the test description here? Maybe "shows the correct number of fairs"?